tsconfig.base.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "typeRoots": ["node_modules/@types"],
```

Comment:
This change fixes `Cannot find namespace 'ApplePayJS'.` TS errors, since it is not possible to import any type from `node_modules/@types/applepayjs` folder.

<img width="884" alt="Screenshot 2024-10-03 at 11 58 04" src="https://github.com/user-attachments/assets/50f6ca48-b65f-486c-b91d-ebcbb6d7a30a">

Reply:
Nice work on doing this 👏